operations/update.go

```diff
@@ -53,12 +53,22 @@ func Update() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
-			return DoUpdate(conf, ctx, doInstall, forceUpdate)
+			if conf.AutoUpgradeCLI {
+				fmt.Println("Automatic CLI updates are already run before each command, get-update command is not necessary")
+				return nil
+			}
+			if !conf.AutoUpgradeCLI && confirm(fmt.Sprint("Automatic CLI updates are not set up, configure now? (Y/n):"), false) {
```

Comment:
Lowercase "y" since "Y" conventionally means "yes to multiple prompts" in a CLI, whereas this is a single prompt.

Rather than make this a prompt, can it be a boolean flag on the command instead? Seems like it would be a hassle to always respond "n" if a user was prompted every time if they want to set up auto-upgrading when they're trying to manually upgrade.